packages/alfa-cli/bin/alfa/command/audit/flags.ts

```diff
@@ -0,0 +1,40 @@
+import { Flag } from "@siteimprove/alfa-command";
+
+import * as scrape from "../scrape/flags";
+
+export const Flags = {
+  help: Flag.help("Display the help information."),
+
+  interactive: Flag.boolean(
+    "interactive",
+    "Whether or not to run an interactive audit."
+  )
+    .alias("i")
+    .default(false),
+
+  format: Flag.string("format", "The reporting format to use.")
+    .type("format or package")
+    .alias("f")
+    .default("earl"),
```

Comment:
Should that have a `.choices` restricting to the formats we support?

Reply:
External formatters can be loaded as well. For example, `--format foo` would look for these packages:

- `alfa-formatter-foo`
- `foo`
- `@siteimprove/alfa-formatter-foo`
- `@siteimprove/foo`